settings/authentication-personalization/authentication-setup/mintlify.mdx

```diff
@@ -1,35 +1,44 @@
 ---
 title: 'Mintlify Dashboard Handshake'
-description: 'Use the Mintlify Dashboard to authenticate users'
+description: 'Use Mintlify to authenticate users'
 ---
 
+<Note>
+Mintlify Dashboard Handshake is available only by request. Please <a href="mailto:sales@mintlify.com">contact our sales team</a> for access. After we've enabled access, you can follow the implementation instructions.
```

Comment:
this is technically true, but a little confusing. All the handshakes are available once you have authentication enabled. I would recommend revising this to
```suggestion
Authentication is available only by request. Please <a href="mailto:sales@mintlify.com">contact our sales team</a> for access. After we've enabled access, you can follow the implementation instructions.
```

Reply:
ah I misunderstood how it works!! pls commit for clarity